docs/guides/blocks/cards.md

```diff
@@ -0,0 +1,224 @@
+
+```ts
+const ProductSchema = coda.makeObjectSchema({
+  properties: {
+    sku: { type: coda.ValueType.String },
+    name: { type: coda.ValueType.String },
+    // ...
+  },
+  // Use the SKU in mention chips, to save space.
+  displayProperty: "sku",
+  // Display the full product name in cards.
+  titleProperty: "name",
+  // ...
+});
+```
+
+
+### Subtitles
+
+The card can include a subtitle that highlights key properties, which appears under the title.
+
+<img src="../../../images/cards_parts_subtitle.png" srcset="../../../images/cards_parts_subtitle_2x.png 2x" alt="The card's subtitle">
+
+The properties displayed in the subtitle are determined via the `subtitleProperties` field of the schema, which lists the subset of properties to show and the order to show them.
+
+```ts
+const ProductSchema = coda.makeObjectSchema({
+  properties: {
+    // ...
+    quantity: { type: coda.ValueType.Number },
+    price: {
+      type: coda.ValueType.Number,
+      codaType: coda.ValueHintType.Currency,
+    },
+  },
+  // ...
+  subtitleProperties: ["quantity", "price"],
+});
+```
+
+The subtitle shows a label for each property, which defaults to `{Name}: {Value}`. You can customize the label for a property, such as removing the property name, in the property definition. See the [Schemas guide][schemas_property_labels] for more information.
+
+
+### Body
+
+The card body can include a snippet of content, which appears under the title (and subtitle if defined).
+
+<img src="../../../images/cards_parts_snippet.png" srcset="../../../images/cards_parts_snippet_2x.png 2x" alt="The card's body">
+
+The snippet is meant to contain a limited amount of text, although there is no size limit enforced. Which property's content to use for the snippet is defined by the field `snippetProperty`, and it can only refer to properties of type `String` or `Array` of `String`. These properties can contain rich text, such as [Markdown][data_types_markdown] and [HTML][data_types_html].
+
+```ts
+const ProductSchema = coda.makeObjectSchema({
+  properties: {
+    // ...
+    description: { type: coda.ValueType.String },
+  },
+  // ...
+  snippetProperty: "description",
+});
+```
+
+
+### Image
+
+The card can include an image, which appears to the left of the other content.
+
+<img src="../../../images/cards_parts_image.png" srcset="../../../images/cards_parts_image_2x.png 2x" alt="The card's image">
+
+Which property's content to use for the image is defined by the field `imageProperty`, and it can only refer to properties of type `String` with a hint of either `ImageReference` or `ImageAttachment`.
```

Comment:
i guess maybe we should omit `ImageAttachment` for now given the stalling of post-processing so it wont work quite properly. 

it would be nice to include a sentence that links out to creating SVGs here as a special call-out of making your own images if desired

Reply:
Done and done.